### ISSUE TITLE
Override --write-all-thumbnails option

### ISSUE THREAD

**Issue by jb2k1** [yt-dlp/yt-dlp#1900]
### Checklist

- [X] I'm reporting a feature request
- [X] I've verified that I'm running yt-dlp version **2021.12.01**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

Currently there is no way to override the `--write-all-thumbnails` option, even with `--no-write-thumbnail`

Example:
user configuration (%appdata%/yt-dlp/config.txt) contains `--write-all-thumbnails`
`yt-dlp.exe --no-write-thumbnail URL` still writes all thumbnails.

Solutions:
add `--no-write-all-thumbnails` option
or
`--no-write-thumbnail` option should disable all thumbnails regardless of `--write-all-thumbnails` option

**Comment by jb2k1** [comment]
Found that `--output "thumbnail:"` does stop thumbnails from being written as a workaround.

**Comment by pukkandan** [comment]
Yes, that does work, but I think it still makes sense for `--no-write-thumbnail` to override `--write-all-thumbnail` like u suggested
